src/slackify/slack.py

```diff
@@ -46,6 +47,6 @@ def block_reply(blocks: list):
 
 
 @async_task
-def respond(url: str, message: str):
+def respond(url: str, message: Dict[str, str]):
```

Comment:
I would leave it as `Dict[str, Any]` as both 
`{'text': 'some text'}` and `{'blocks': [...]}` are valid inputs for this function

Reply:
Absolutely, makes total sense. Only saw the example I was working on. Thanks, will push an update!